src/GeneratedMapper/MapperGenerator.cs

```diff
@@ -21,6 +22,12 @@ public class MapperGenerator : ISourceGenerator
         public void Initialize(GeneratorInitializationContext context)
         {
             context.RegisterForSyntaxNotifications(() => new MapAttributeReceiver());
+#if DEBUG
```

Comment:
This was a cool thing I saw where you can debug the source generator against an application in real time.

Reply:
Ah that's is cool. I've struggled to debug the source generator and that's why the unit tests are more like integration tests fully testing the entire generator, instead of the individual units.